app/views/shipit/tasks/new.html.erb

```diff
@@ -13,13 +13,32 @@
 
   <%= render_checklist @task.checklist %>
 
-  <section class="submit-section">
-    <%= form_for @task, url: stack_tasks_path(@stack, definition_id: @task.definition.id) do |f| %>
+  <%= form_for @task, url: stack_tasks_path(@stack, definition_id: @task.definition.id) do |f| %>
```

Comment:
I've added some changes after discovering a bug while doing functional tests locally but I squashed my commits by mistake so I'll comment on what they are.

I've added a form in the UI here so that the task can be triggered from shipit